jobs/forms.py

```diff
@@ -19,3 +19,18 @@ class Meta:
             'title': 'Job title'
         }
 
+
+class MeetupForm(forms.ModelForm):
+
+    class Meta:
+        model = Meetup
+        exclude = ['reviewer', 'review_status', 'reviewers_comment',
+            'ready_to_publish', 'published_date', 'created', 'expiration_date'
+        ]
+
```

Comment:
@ksiedlarek actually I didn't look at your whole form and a template, just on the @rebkwok comment which mentioned me. Now, I see what you aiming at.
I would need to think about it a bit and maybe run it locally, but generally I'm not very fond of the dropdown and a separate form to add the company. It somehow always annoys me when people force me to do things like that. And yet, there is this problem with the Integrity Error and we need to think what is the best way to solve it. In any case, we should use the same mechanism for the Add Job and the Add Meetup point. I would be confusing otherwise and also it will be harder to maintain.

As for the exclude - Two Scoops of Django advises not to use 'exclude'. I don't have it on me right now but when I'm back home, I can paste a quote or something.